packages/edit-site/src/store/selectors.js

```diff
@@ -98,6 +98,8 @@ export const getSettings = createSelector(
 			hasFixedToolbar: isFeatureActive( state, 'fixedToolbar' ),
 			__experimentalSetIsInserterOpened: setIsInserterOpen,
 			__experimentalReusableBlocks: getReusableBlocks( state ),
+			__experimentalPrioritizePatternsOnQuickInserterRoot:
```

Comment:
Nit: I'd be filter for this to be stable `preferPatternsOnRoot` or something but I'm ok if you think we should start experimental.